scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml

```diff
@@ -0,0 +1,199 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: libvirt
+      libvirt:
+        hypervisor_ip: ""
```

Comment:
Hrm, no, it's not supposed to be like this (see change I did in qa_crowbarsetup). I guess I lost a change here... Will fix.